### PR TITLE
Upgrade ShadowsocksX to 'forever' latest

### PR DESCRIPTION
**Pull request by viztor**

### CODE REVIEW

File: Casks/shadowsocksx.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 cask :v1 => 'shadowsocksx' do
-  version '2.6.1'
-  sha256 '79650f9c592b69b0cbe867962f2aeedfcd07623c12db9585b533e7c5786c6d6f'
+  version :latest
+  sha256 :no_check
 
-  url "http://downloads.sourceforge.net/project/shadowsocksgui/dist/ShadowsocksX-#{version}.dmg"
+  url "http://sourceforge.net/projects/shadowsocksgui/files/latest/download"
```
Comment (vitorgalvao) [inline Casks/shadowsocksx.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

**Comment by fanquake** [comment]
@viztor Can you squash your commits?

File: Casks/shadowsocksx.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 cask :v1 => 'shadowsocksx' do
-  version '2.6.1'
-  sha256 '79650f9c592b69b0cbe867962f2aeedfcd07623c12db9585b533e7c5786c6d6f'
+  version :latest
+  sha256 :no_check
 
-  url "http://downloads.sourceforge.net/project/shadowsocksgui/dist/ShadowsocksX-#{version}.dmg"
+  url 'http://sourceforge.net/projects/shadowsocksgui/files/latest/download'
```
Comment (vitorgalvao) [inline Casks/shadowsocksx.rb]: Before you [squash your commits](http://davidwalsh.name/squash-commits-git), please add a `# sourceforge.net is the official download host per the vendor homepage` line above this one.

**Comment by ndr-qef** [comment]
Merged as a9abcce. Thanks.

**Pull request closed without merging**